release/README.adoc

```diff
@@ -124,3 +124,67 @@ flowchart LR
 ----
 
 Note that the configuration of CircleCI *disallow* multiple build on the same branch (a new build cancel the previous one). It means that we have to run one step at a time.
+
+
+== 🏁 Run release alpha version pipeline
+
+Lunch only required steps in one CircleCI pipeline to release an alpha version. There are two methods available:
+
+* npm run
+[source, shell]
+----
+npm run am-release -- \
+  --branch master --gio-version 4.3.0 \
+  --dry-run \
+  release-version
+----
+* curl HTTP
+[source, shell]
+----
+curl --location --request POST 'https://circleci.com/api/v2/project/gh/gravitee-io/gravitee-access-management/pipeline' \
+--header 'Content-Type: application/json' \
+--header "Circle-Token: $CIRCLECI_TOKEN" \
+--data-raw '{
+    "branch": "4.3.x",
+    "parameters": {
+        "gio_action": "release-alpha-version",
+        "dry_run": true,
+        "rc_requested": false,
+        "graviteeio_version": "4.3.3-alpha",
```

Comment:
and here. Same for `release-version`

Reply:
there is the next minor version for 4.3.x